index.html

```diff
@@ -50,6 +52,250 @@ <h1>Hey there. I’m Mariane<br />
+                                <p class="tech-name">html</p>
+                            </li>
+                        </ol>
+                        <button class="btn btn-projects"  type="button">See Projects</button>
+                    </div>
+                </div>
+                <div class="project-container-card">
+                    <div class="project-image"></div>
+                    <div class="project-desc-tech-details">
+                        <h4 class="project-desc">Multi-Post Stories<br /> Gain + Glory</h4>
+                        <ol class="tech-list-container">
+                            <li class="tech-list">
+                                <p class="tech-name">Ruby on Rails</p>
+                            </li>
+                            <li class="tech-list">
+                                <p class="tech-name">css</p>
+                            </li>
+                            <li class="tech-list">
+                                <p class="tech-name">JavaScript</p>
+                            </li>
+                            <li class="tech-list">
+                                <p class="tech-name">html</p>
+                            </li>
+                        </ol>
+                        <button class="btn btn-projects"  type="button">See Projects</button>
+                    </div>
+                </div>
+                <div class="project-container-card">
+                    <div class="project-image"></div>
+                    <div class="project-desc-tech-details">
+                        <h4 class="project-desc">Multi-Post Stories<br /> Gain + Glory</h4>
+                        <ol class="tech-list-container">
+                            <li class="tech-list">
+                                <p class="tech-name">Ruby on Rails</p>
+                            </li>
+                            <li class="tech-list">
+                                <p class="tech-name">css</p>
+                            </li>
+                            <li class="tech-list">
+                                <p class="tech-name">JavaScript</p>
+                            </li>
+                            <li class="tech-list">
+                                <p class="tech-name">html</p>
+                            </li>
+                        </ol>
+                        <button class="btn btn-projects"  type="button">See Projects</button>
+                    </div>
+                </div>
+            </div>
+            <div class="about-me">
+                <h1>About me</h1>
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+                <button class="btn btn-projects"  type="button">Get my resume</button>
+            </div>
```

Comment:
- [ ] In your about-me `div` you forgot to add a background image. Let's fix it by adding a proper background image using CSS. TIP 📌: Use `background-image: ...;` property in CSS as you have done in your hero section 🤓🍀
#### Your design
<img width="326" alt="Screen Shot 2022-01-15 at 3 10 28 PM" src="https://user-images.githubusercontent.com/69011963/149618209-35dd210a-ddaf-4299-836e-8073799e31cc.png">

#### Figma design
<img width="238" alt="Screen Shot 2022-01-15 at 3 16 26 PM" src="https://user-images.githubusercontent.com/69011963/149618229-81ef138b-0c46-462a-bd9e-d528660eac8e.png">

Reply:
Well implemented 👍🏽

Reply:
Hello @Rustamxon7 ,
Thanks for your review 😄 . I have added the background image in the about-me section